### ISSUE TITLE
filter issue

### ISSUE THREAD

**Issue by Mwamini** [freeCodeCamp/freeCodeCamp#13247]
Please describe how to reproduce this issue, and include links to screenshots if possible.
It does accept filter, i've checked my codes everything seems perfect but it does not run
My code:
```javascript

function sumFibs(num) {
  var list= fibUpTo(num);
  
  return list.filter(function(number){return number % 2===1;})
  .reduce(function(p,c){return p + c;}, 0);
 
}

function fibUpTo(num){
  if(num===1){
    var list=[1, 1];
    while(true){
      var next=list(list.length-1) + list(list.length-2);
      if(next<=num){
        list.push(next);
      }
      else {
        return list;
      }
    }
   
  }
}
sumFibs(4);

```

**Comment by atjonathan** [comment]
Your code is incorrect. Please use the [Help](https://gitter.im/FreeCodeCamp/Help) chat room for getting challenge related help. Happy Coding!

**Comment by Mwamini** [comment]
Thanks Jonathan.

Sent from Yahoo Mail on Android 
 
  On Wed, Feb 8, 2017 at 2:21 PM, Jonathan<notifications@github.com> wrote:   
Closed #13247.

**Comment by atjonathan** [comment]
@Mwamini, no problem 😀
